src/infra/manifests/yaml/reader.go

```diff
@@ -86,3 +91,15 @@ func (r *Reader) buildMessages(fp string) ([]*manifest.Manifest, error) {
 
 	return mnfs, nil
 }
+
+func fillMap(mnfs []entities.Manifest, manifestsMap map[string][]entities.Manifest) {
```

Comment:
Minor: Method name is not very descriptive, I would suggest something like:
```
func aggregateManifestKind(mnfs []entities.Manifest) map[string][]entities.Manifest {
....
```

Reply:
done